htdocs/webportal/public/controllers/propallist.controller.php

```diff
@@ -0,0 +1,115 @@
+<?php
```

Comment:
Php filenames must be
.class.php for classes
.inc.php for include of pages.
Here the file is a class (it extends another class), so it must end with .class.php

Reply:
It has been done